tools/python/influx-migration/influx_migration.py

```diff
@@ -119,14 +119,15 @@ def report_all_bucket_series_count(host, token, org_name=None):
         while len(buckets.buckets) > 0:
             for bucket in buckets.buckets:
                 if not bucket.name.startswith("_"):
-                    report_bucket_series_count(bucket_name=bucket.name, host=host, token=token, org_name=org_name)
+                    report_bucket_series_count(bucket_name=bucket.name, host=host, token=token,
+                        org_name=org_name, skip_verify=skip_verify)
             offset += BUCKET_PAGINATION_LIMIT
             buckets = client.buckets_api().find_buckets(limit=BUCKET_PAGINATION_LIMIT,
                 offset=offset)
 
-def report_bucket_series_count(bucket_name, host, token, org_name=None):
+def report_bucket_series_count(bucket_name, host, token, org_name=None, skip_verify=False):
```

Comment:
Can the last parameter to this method be "verify_ssl" that you can pass in the value as-is to  InfluxDBClient()?

Reply:
`skip_verify` originates from the CLI argument `--skip-verify`. This is also what the Influx CLI uses. The `--skip-verify` argument is passed essentially as-is to all places the Influx CLI is used.

This means that the value cannot be currently passed in as is, since the value will originate as `--skip-verify`, having the opposite meaning of `--verify-tls`.

We could instead change the CLI option from `--skip-verify` to `--verify-tls`, allowing the value to be passed as is. But then the value could not be passed to the Influx CLI as is, and would have to be negated.